src/client/pps.go

```diff
@@ -329,116 +383,88 @@ func (c APIClient) ListJobFilterF(pipelineName string, inputCommit []*pfs.Commit
 	}
 }
 
-// FlushJob calls f with all the jobs which were triggered by commits.
-// If toPipelines is non-nil then only the jobs between commits and those
-// pipelines in the DAG will be returned.
-func (c APIClient) FlushJob(commits []*pfs.Commit, toPipelines []string, f func(*pps.JobInfo) error) error {
-	req := &pps.FlushJobRequest{
-		Commits: commits,
-	}
-	for _, pipeline := range toPipelines {
-		req.ToPipelines = append(req.ToPipelines, NewPipeline(pipeline))
-	}
-	client, err := c.PpsAPIClient.FlushJob(c.Ctx(), req)
+// SubscribeJob calls the given callback with each open job in the given
+// pipeline until canceled.
+func (c APIClient) SubscribeJob(pipelineName string, includePipelineInfo bool, cb func(*pps.JobInfo) error) error {
```

Comment:
It seems like `includePipelineInfo` should just be called `full`. Also, is "open job" what we are calling a job that isn't terminal?

Reply:
I think 'open' job works, but I'm open to using a different term.